dynatrace-operator-google-marketplace/chart/dynatrace-operator/values.yaml

```diff
@@ -67,7 +73,13 @@ kubernetesMonitoring:
   customProperties:
     value: ""
     valueFrom: ""
-  resources: { }
```

Comment:
Any reason to remove this line? It's on the non-GKE values.yaml as well. And the example below can remain as well.

Reply:
We need to drop this here otherwise creating it via the UI does not work as expected

Reply:
Got it.